client/src/components/framework/toasters.ts

```diff
@@ -26,8 +26,10 @@ export const keepAroundErrorToast = (message: any) =>
 /*
 a hard network error
 */
-// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any -- - FIXME: disabled temporarily on migrate to TS.
-export const postNetworkErrorToast = (message: any, key = undefined) =>
+export const postNetworkErrorToast = (
+  message: string,
+  key: string | undefined = undefined
```

Comment:
I think this can just be:
```suggestion
  key?: string
```

Reply:
@seve let's change it to `key?: string` 🎉 !